File: test/data/phenology/crop2ml/algo/cs/PhylSowingDateCorrection.cs

if (latitude < 0.0)
{
	if (sowingDay > sDsa_sh)
	{
		fixPhyll = p * (1 - rp * Math.Min(sowingDay - sDsa_sh, sDws));
	}
	else fixPhyll = p;
}
else
{
	if (sowingDay < sDsa_nh)
	{
		fixPhyll = p * (1 - rp * Math.Min(sowingDay, sDws));
	}
	else fixPhyll = p;
}

File: test/data/phenology/crop2ml/algo/f90/PhylSowingDateCorrection.f90

    !use crop2mlModules

        IF (latitude < 0) THEN
            IF (sowingDay > sDsa_sh) THEN
                fixPhyll = p * (1 - rp * MIN(sowingDay - sDsa_sh, sDws))
            ELSE
                fixPhyll = p
            END IF
        ELSE
            IF (sowingDay < sDsa_nh) THEN
                fixPhyll = p * (1 - rp * MIN(sowingDay, sDws))
            ELSE
                fixPhyll = p
            END IF
        END IF



